kolibri_explore_plugin/assets/src/views/ChannelsPage.vue

```diff
@@ -114,4 +133,13 @@
     background: linear-gradient(to bottom, rgba($black, 0.075) 0, rgba($black, 0) 5px) #dee2e6;
   }
 
+  .header {
+    background: #dee2e6;
+    border-bottom: 1px solid #cccccc;
```

Comment:
Please use colors from the SCSS theme.

Reply:
ups, I got that values from figma, but I didn't notice that we've that values as bootstrap variables.

Reply:
No problem! Great, we should always use the theme so we can change everything in the same single place. If we had a CI check for hardcoded colors I would already enabled it hehe.